src/planner.c

```diff
@@ -65,6 +66,7 @@ static set_rel_pathlist_hook_type prev_set_rel_pathlist_hook;
 static get_relation_info_hook_type prev_get_relation_info_hook;
 static create_upper_paths_hook_type prev_create_upper_paths_hook;
 static bool contain_param(Node *node);
+static void pushdown_orderby_to_groupclause(Query *query);
```

Comment:
i think the naming should be consistent, so this should be reorder_groupby as well